lib/services/delivery_service.dart

```diff
@@ -21,20 +23,69 @@ class DeliveryService {
   ///  collection and `null` if no delivery is found with the specified
   ///  criteria.
   Future<Delivery?> getDelivery(DateTime date, String donorId) async {
-    final deliveryQuerySnapshot = await _deliveryCollection
-        .where('pickUpFrom', isEqualTo: date)
-        .where('donorId', isEqualTo: donorId)
-        .get();
-
-    if (deliveryQuerySnapshot.docs.isNotEmpty) {
-      return deliveryQuerySnapshot.docs.first.data();
-    }
+    // TODO: Add DeliveryDto to Delivery mapping and return correct instance
```

Comment:
Why do we have this TODO? Can we add a link to the JIRA ticket in which this will be addressed?

Reply:
Yea, actually there was no JIRA ticket for that one when I have added this TODO 😅 

But it is already addressed in the last PR:

https://github.com/zachran-jidlo/zachranobed/pull/77/files#diff-282ca8dace2f43785f1fc8c96c398e38f56b3b59047e93fa2c3e350961b32c81L33

![image](https://github.com/zachran-jidlo/zachranobed/assets/80771513/e63cf397-8452-417e-ab80-bc99475af376)

Is it OK for you?

Reply:
Yes. Let's merge it.